src/pages/costModels/costModel/priceListTable.tsx

```diff
@@ -306,29 +276,6 @@ class PriceListTable extends React.Component<Props, State> {
                       ]}
                       tiers={filtered}
                     />
-
-                    <Toolbar id="price-list-toolbar-bottom">
-                      <ToolbarContent>
-                        <ToolbarItem variant={ToolbarItemVariant.pagination}>
-                          <Pagination
-                            itemCount={res.length}
-                            perPage={this.state.pagination.perPage}
-                            page={this.state.pagination.page}
-                            onSetPage={(_evt, page) =>
-                              this.setState({
-                                pagination: { ...this.state.pagination, page },
-                              })
-                            }
-                            onPerPageSelect={(_evt, perPage) =>
-                              this.setState({
-                                pagination: { page: 1, perPage },
-                              })
-                            }
-                            variant="bottom"
-                          />
-                        </ToolbarItem>
-                      </ToolbarContent>
-                    </Toolbar>
```

Comment:
Trying to find where this was used in the UI? For example, the cost model wizard still has pagination for rates.

Reply:
ah, I'll take a look

Reply:
it looks like the cost model wizard has it's own implementation..  https://github.com/project-koku/koku-ui/blob/main/src/pages/costModels/createCostModelWizard/priceListTable.tsx#L205